types/office-js-preview/index.d.ts

```diff
@@ -24368,5615 +24368,5615 @@ declare namespace Excel {
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         PNG = "PNG",
         /**
-        * The TIFF file type.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * The TIFF file type.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         TIF = "TIF",
         /**
-        * The WEBP file type.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * The WEBP file type.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         WEBP = "WEBP"
     }
     /**
-    * Represents a card layout that is best used for an array.
-    *
-    * @remarks
-    * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-    * @beta
-    */
+     * Represents a card layout that is best used for an array.
+     *
+     * @remarks
+     * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
     interface EntityArrayCardLayout {
         /**
-        * Represents the type of this layout.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * Represents the type of this layout.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         layout: EntityCardLayoutType.array | "Array";
         /**
-        * Represents name of the property that contains the array shown in the card.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * Represents name of the property that contains the array shown in the card.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         arrayProperty: string;
         /**
-        * Represents name of the property that contains the array shown in the card. Default value is "Array".
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * Represents name of the property that contains the array shown in the card. Default value is "Array".
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         displayName?: string;
         /**
-        * Represents the count of rows which the card claims are in the array. A card may report a different number of rows than it actually has to display smaller amounts of preview data.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * Represents the count of rows which the card claims are in the array. A card may report a different number of rows than it actually has to display smaller amounts of preview data.
```

Comment:
The 2nd sentence is ungrammatical. I can't tell what it means.

Reply:
Thanks for the review! I'll fix these.